### ISSUE TITLE
Release ?

### ISSUE THREAD

**Issue by aelmahmoudy** [microg/OpenWlanMapNlpBackend#1]
Hello, any reason this hasn't been released yet ?

**Comment by Mosrite** [comment]
There was a version on F-Droid, but it seems to be gone. I wonder what happened?

**Comment by aelmahmoudy** [comment]
The build is disabled until a release is made

**Comment by mar-v-in** [comment]
I wanted to rewrite the communication with openwlanmap.org servers as libwlocate is totally messy and does not integrate well with UnifiedNlp, but I had no time to do this yet...

**Comment by frazzmark** [comment]
@mar-v-in yeah, libwlocate is really messy, with no debug and a strange working logic.
If you do rewrite it, could you please upload the code to a separate project?
